lib/octocatalog-diff/cli/options/storeconfigs_backend.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Set storeconfigs (integration with PuppetDB for collected resources)
+# @param parser [OptionParser object] The OptionParser argument
+# @param options [Hash] Options hash being constructed; this is modified in this method.
+OctocatalogDiff::Cli::Options::Option.newoption(:storeconfigs) do
+  has_weight 220
```

Comment:
What does has weight do?

Reply:
The options class requires that a weight be specified, so alllllll of the options defined in this code have a relative weight.  It's not going to affect anything negatively here, since it's using the same weight as the storeconfigs.rb value, and there won't be any other option to compare value against.